src/pages/swap/index.tsx

```diff
@@ -280,7 +305,7 @@ export const SwapPage = () => {
           <a
             href={firstSep24ResponseState.url}
             target="_blank"
-            rel="noreferrer"
+            rel="opener" //noopener forbids the use of postMessages.
```

Comment:
```suggestion
            rel="opener" //noreferrer forbids the use of postMessages.
```

Reply:
I wasn't saying it does, I only suggested a change for your comment because I thought you meant to mention `noreferrer` here 😅  I checked the difference between `noopener` and `noreferrer` again and they both achieve the same in this case (both would forbid using postMessage()) so nevermind my suggestion.